staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go

```diff
@@ -190,6 +190,14 @@ func IsValidPortNum(port int) []string {
 	return []string{InclusiveRangeError(1, 65535)}
 }
 
+// IsValidPortNumWithZeroInclusive tests that the argument is a valid port number, with zero inclusive.
+func IsValidPortNumWithZeroInclusive(port int) []string {
```

Comment:
This is not really generically useful, can we instead just check "or equals 0" at the callsite?